README.md

```diff
@@ -20,6 +20,8 @@ You can read more about our values [here](/docs/community-values-en.md).
 
 You can submit a talk or a workshop proposal [here](https://github.com/parisp2p/community/wiki/Call-For-Proposal).
 
+See [current proposals](https://github.com/orgs/parisp2p/projects/1).
```

Comment:
https://github.com/orgs/parisp2p/projects/1 lead to a 404, should we just point to the listing (https://github.com/orgs/parisp2p/projects/) instead ?

Reply:
bad link :) 

the good one is: https://github.com/orgs/francep2p/projects/1

will update the PR